packages/wonder-blocks-dropdown/src/util/types.ts

```diff
@@ -63,6 +63,11 @@ export type MaybeValueOrValues = MaybeString | Array<MaybeString>;
  * The labels for the combobox component.
  */
 export type ComboboxLabels = {
+    /**
+     * Label for the "Clear" button that removes a selected item. Only used in
+     * single-select mode.
```

Comment:
question: Is there a reason we can't use it with the multiple selection type?

Reply:
Right! The clear button for multi-selection could be helpful for a "Clear all" functionality, especially if there are many pills selected! This could be a future improvement though if it's needed 😄